### ISSUE TITLE
Can't read the last message on the queue while using Apache ActiveMQ Artemis 2.16.0

### ISSUE THREAD

**Issue by piroMe** [jmstoolbox/jmstoolbox#101]
## Specifications
- JMSToolBox version:
- OS (Name, version, 32/64bits):
- Q provider (kind, version) :

## Steps to Reproduce the Problem

  1.
  1.
  1.

## Logs
Eventually, attach the log files which are `"<user>\.jtb\JMSToolBox\jmstoolbox.log"` and `"<user>\.jtb\.metadata\.log"`    
They can be exported directly from JMStoolBox: Menu "Help" > "Export JMSToolBox log files..."

**Comment by titou10titou10** [comment]
The logs updated does not contain the stack trace    
Q: do you have other program than JTB that can read the messages?    
It seems you are in this situation: https://stackoverflow.com/questions/59651341/polling-byte-large-messages-from-activemq-artemis-using-apache-camel-consumert    
Could you try to add`?minLargeMessageSize=<some large value ie 512MiB or more>`to your brokerURL? 
Ref: https://activemq.apache.org/components/artemis/documentation/latest/large-messages.html

**Comment by piroMe** [comment]
do you have other program than JTB that can read the messages?

No.

what is the size of your messages?

varies in size [3kb to 1GB]

Could you try to add?minLargeMessageSize=<some large value ie 512MiB or more>to your brokerURL

how this could help?

**Comment by titou10titou10** [comment]
> **how this could help?**

- The answer is in the second link of my previous post that.  Did you read it?
- Did you try it, ie set minLargeMessageSize with a value bigger that your biggest message?

"Large message" is a very specific proprietary "feature" in ActiveMQ and Artemis . Client has to be aware of large message and process them differently wich is not the vocation of a tool like JTB that is a generic JMS tool. Setting this parameter will make the message not be considered as "large"

**Comment by piroMe** [comment]
yes I've read it and applied your proposed solution, but it did not work.

**Comment by titou10titou10** [comment]
what string do you use for`brokerURL`now?

**Comment by piroMe** [comment]
tcp://localhost:61616?minLargeMessageSize=5368709120
